packages/story-editor/src/components/canvas/editLayer.js

```diff
@@ -90,6 +100,24 @@ function EditLayerForElement({ element, showOverflow }) {
     return () => focusCanvas(/* force */ false);
   }, [focusCanvas]);
 
+  const moveable = useRef(null);
+  const setRef = useCallback(
+    (moveableRef) => {
+      moveable.current = moveableRef;
+      onMoveableMount?.(moveableRef);
+    },
+    [onMoveableMount]
+  );
+
+  const onResize = () => {
```

Comment:
If we don't wrap this into `useCallback`, it will make the `EditElement` always re-render, maybe add `useCallback` here as well?

Reply:
It's not memoized so it will always re-render regardless. We could memoize it of course.

Reply:
I memoized both the component and the callback in 4a44d28bb. But I actually don't think it matters. I cannot think of an instance, where the `editLayer` renders without the `editElement` having to re-render too anyway. So either both have to render or neither, making memoization irrelevant. But memoizing is good practice anyway, so added.